posts/2024-08-27-quarto-on-HPC/index.qmd

```diff
@@ -0,0 +1,166 @@
+---
+title: "Quarto on HPC"
+image: https://quarto.org/quarto.png
+lightbox: true
+description: "A tutorial for setting up Quarto in a remote server (HPC)"
+author:
+  - name: Fernando Cervantes
+    url: https://www.github.com/fercer
+    orcid: 0000-0003-0652-2750
+    affiliation: The Jackson Laboratory
+    affiliation-url: https://jax.org/ 
+date: 2024-08-27
+categories: [HPC,Tutorials] # self-defined categories
+draft: false # setting this to `true` will prevent your post from appearing on your listing page until you're ready!
+---
+
+# What is Quarto?
+
+From the [Quarto](https://quarto.org) webpage, it is
+
+> An open-source scientific and technical publishing system
+
+Quarto can be used to create documentation pages, scientific notebooks, professional presentations, and websites (like this one!).
+
+This system is useful for creating notes, particularly these containing Python/R `code`.
+And in contrast with jupyter notebooks, it offers a wide variety of ways to share these notes.
+
+# Install Quarto on a remote server (HPC)
+
+In this post, two options are presented to install Quarto on a HPC server.
+The first option installs Quarto on your home directory and works fine when creating code-less documents.

```

Comment:
```suggestion
The first option installs Quarto in your home directory and works fine when creating code-less documents.

```

Reply:
I blame English and its inconsistent use of `on` and `in`!!!